app/src/main/java/org/wikipedia/readinglist/ReadingListsFragment.java

```diff
@@ -124,8 +125,8 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         searchEmptyView.setEmptyText(R.string.search_reading_lists_no_results);
         readingListView.setLayoutManager(new LinearLayoutManager(getContext()));
         readingListView.setAdapter(adapter);
-        readingListView.addItemDecoration(new DrawableItemDecoration(requireContext(), R.attr.list_separator_drawable, true, false));
-
+        readingListView.addItemDecoration(new DrawableItemDecoration(requireContext(), R.attr.list_separator_drawable, false, true));
```

Comment:
The top divider will still show up if there's no other reading list than "Saved".

Reply:
Good one @cooltey .. it needed a general fix to all recyclerviews.. addressed here #1369